internal/telemetry/telemetry_test.go

```diff
@@ -19,35 +19,39 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestProductChange(t *testing.T) {
+func TestProductEnabled(t *testing.T) {
 	client := new(client)
 	client.start(nil, NamespaceTracers)
-	client.productChange(NamespaceProfilers, true,
-		[]Configuration{BoolConfig("delta_profiles", true)})
-
+	client.productEnabled(NamespaceProfilers)
 	// should contain app-client-configuration-change and app-product-change
```

Comment:
Is this comment right? it checks for length 1 but says two events? Looks like it's only looking for the app-product-changed?

Reply:
Ah the comment is not right, previously `productEnabled` enqueued client config change info but that work is done in the `configChange` method now -- i'll update this comment